### PR TITLE
HTCONDOR-1312 apply JOB_TRANSFORM_* rules in condor_transform_ads

### PR DESCRIPTION
**Pull request by johnkn**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [x] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [x] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [x] Check for documentation, if needed
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/man-pages/condor_transform_ads.rst [hunk 1]
```diff
@@ -39,6 +43,9 @@ Options
  **-rules** *rules-file*
     Specifies the file containing definitions of the transformation
     rules.
+ **-jobtransforms** *name-list*
+    Specifies one or more transform names.  The transform rules will be read
+    from the configuration file ``JOB_TRANSFORM_<name>`` values.
```
Comment (Todd-L-Miller) [inline docs/man-pages/condor_transform_ads.rst:48]: Seems like you should mention the format of _name-list_ here.  Maybe

```suggestion
 **-jobtransforms** *name-list*
    A comma-separated list of one or more transform names.  The transform rules will be read
    from the configuration file ``JOB_TRANSFORM_<name>`` values.
```
?

File: docs/man-pages/condor_transform_ads.rst [hunk 2]
```diff
@@ -10,23 +10,27 @@ Synopsis
...
 [**-xml** ] [**-verbose** ] [**-terse** ] [**-debug** ]
 [**-unit-test** ] [**-testing** ] [**-convertoldroutes** ] [*infile1
 ...infileN* ]
 
-Note that exactly one rules file, and at least one input file, must be
-specified. If no output file is specified, output will be written to
+Note that one or more transform must be specified in the form of a rules
+file or a ``JOB_TRANSFORM_`` name and at least one input file must be
+specified. Transforms will be applied in the order they are given on the command
+line.  If a complex TRANSFORM statement is used it must be at the end of the last
```
Comment (Todd-L-Miller) [inline docs/man-pages/condor_transform_ads.rst:24]: What is "a complex TRANSFORM statement"?  (If it's defined elsewhere in the manual, that's fine, but I couldn't find it.)

Comment (Todd-L-Miller) [reply docs/man-pages/condor_transform_ads.rst:24]: I think the code calls these "iterating" transforms?

Comment (johnkn) [reply docs/man-pages/condor_transform_ads.rst:24]: yeah. iterating transforms

Comment (Todd-L-Miller) [reply docs/man-pages/condor_transform_ads.rst:24]: I don't actually know what those are, but if people who write transforms do, maybe just say that instead?

**Review by Todd-L-Miller — APPROVED** [review]

**Pull request merged**